docs/get-support.md

```diff
@@ -4,22 +4,33 @@ title: Get Support
 
 # Get Support
 
-The best way to request support from the Event Bus Team is to reach out to us via the [#ves-event-bus](https://dsva.slack.com/archives/C042ZQ7JUAX) Slack channel in the Office of CTO @VA workspace.
+## General support requests
 
-Please avoid direct messaging an Event Bus team member in Slack. Chatting openly in the [#ves-event-bus](https://dsva.slack.com/archives/C042ZQ7JUAX) channel allows others to learn from the questions and answers discussed.
+The best way to request support from the Event Bus Team is to reach out to us via the [#event-bus-support][slack-#event-bus-support] Slack channel in the Office of CTO @VA workspace.
 
-## Slack channel support hours 
+Please avoid direct messaging an Event Bus team member in Slack. Chatting openly in the [#event-bus-support][slack-#event-bus-support] channel allows others to learn from the questions and answers discussed.
 
-Event Bus Slack channel support hours are from 9am - 5pm ET, Monday through Friday, with the exception of federal holidays.
+Event Bus Slack channel support hours are from 12pm - 5pm ET, Monday through Friday, with the exception of federal holidays. Often, an engineer will respond by 9am ET if they live in the Eastern or Central timezones. You can expect a response within 1 hour during the support hours.
 
-Response times are based on the nature of the posting:
+## Report an incident
```

Comment:
Not sure where it would fall in, but I think it is worth calling out our [incident-response](https://dsva.slack.com/archives/C06LG96V4BW) channel as I imagine we will begin using that channel as more folks onboard